src/encoded/schemas/ivp_d1.json

```diff
@@ -0,0 +1,2148 @@
+                                    "type": "string",
+                                    "enum": [
+                                        "0 No",
+                                        "1 Yes"
+                                    ]
+                                },
+                                "mcin2att": {
+                                    "title": "5d2. Attention",
+                                    "type": "string",
+                                    "enum": [
+                                        "0 No",
+                                        "1 Yes"
+                                    ]
+                                },
+                                "mcin2ex": {
+                                    "title": "5d3. Executive",
+                                    "type": "string",
+                                    "enum": [
+                                        "0 No",
+                                        "1 Yes"
+                                    ]
+                                },
+                                "mcin2vis": {
+                                    "title": "5d4. Visuospatial",
+                                    "type": "string",
+                                    "enum": [
+                                        "0 No",
+                                        "1 Yes"
+                                    ]
+                                }
+                            }
+                        },
+                        "impnomci": {
+                            "title": "5e. Cognitively impaired, not MCI",
+                            "type": "string",
+                            "enum": [
+                                "1 Present",
+                                "0 Absent"
+                            ]
+                        }
+                    }
+                }
+            }
+        },
+        "sec2": {
+            "title": "SECTION 2: Biomarkers, imaging, and genetics",
+            "description": "Section 2 must be completed for all subjects.",
+            "type": "object",
+            "properties": {
+                "biomarker_find": {
+                    "title": "6. Indicate neurodegenerative biomarker status, using local standards for positivity.\nBiomarker Findings",
+                    "type": "object",
+                    "properties": {
+                        "note_d1_11": {
+                            "title": "Section 2 must be completed for all subjects.",
+                            "type": "string"
+                            
+                        },
+                        "amylpet": {
+                            "title": "6a. Abnormally elevated amyloid on PET",
+                            "type": "string",
+                            "enum": [
+                                "No 0",
+                                "Yes 1",
+                                "Unknown, not assessed 8"
+                            ]
```

Comment:
for radio button, check the design book for numbering.
![image](https://user-images.githubusercontent.com/44071821/99461216-0290a880-28f7-11eb-89b0-f5302c823b35.png)

Reply:
This one I talked to Venkat today, for mapping, if you prefer to put the number first, it's better we change the redcap version with put number first. So the option showing in page should be 
0 No, 1 Yes, 8 Unknown, not assessed
won't be:  0, No 0, 1, Yes 1, 8 Unknow, not assessed 8...

![image](https://user-images.githubusercontent.com/38848214/99463217-decf6180-28fa-11eb-8198-81916f78222c.png)
![image](https://user-images.githubusercontent.com/38848214/99463230-e858c980-28fa-11eb-9f6e-a7636d651ed6.png)
![image](https://user-images.githubusercontent.com/38848214/99463254-f73f7c00-28fa-11eb-8d76-d4824e1e4659.png)
![image](https://user-images.githubusercontent.com/38848214/99463272-00304d80-28fb-11eb-956d-66834bffb62c.png)